python/ray/data/tests/test_formats.py

```diff
@@ -185,12 +186,32 @@ def test_from_torch(shutdown_only, tmp_path):
     torch_dataset = torchvision.datasets.MNIST(tmp_path, download=True)
     expected_data = list(torch_dataset)
 
-    ray_dataset = ray.data.from_torch(torch_dataset)
+    ray_dataset = ray.data.from_torch(torch_dataset, parallelism=1)
 
     actual_data = extract_values("item", list(ray_dataset.take_all()))
     assert actual_data == expected_data
 
 
+def test_from_torch_parallel(shutdown_only, tmp_path):
+    # parallel read may reorder blocks, cannot directly assert equality on outputs
+    torch_dataset = torchvision.datasets.MNIST(tmp_path, download=True)
+    expected_data = list(torch_dataset)
+
+    ray_dataset = ray.data.from_torch(torch_dataset, parallelism=3)
```

Comment:
do we need to explicitly specify parallelism here?

Reply:
As long as the parallelism autodetection parallelizes when possible,  I guess not